flocker/control/test/test_protocol.py

```diff
@@ -50,6 +54,26 @@
 from .clusterstatetools import advance_some, advance_rest
 
 
+def random_ip():
+    return inet_ntop(AF_INET6, urandom(16))
+
+
+def arbitrary_transformation(deployment):
+    """
+    Make some change to a deployment configuration.  Any change.
+
+    The exact change made is unspecified but the resulting ``Deployment`` will
+    be different from the given ``Deployment``.
```

Comment:
The change is specified though, a new empty node is added to the deployment, so the function name and docstring should be adjusted. Also I don't think you need `random_ip` because you don't need to specify a hostname when you specify a UUID.